### PR TITLE
PEN-1237/update alc documentation

### PR DESCRIPTION
**Pull request by RobinGiannattasio**
## Description
_This PR updates the user facing ALC documentation for the Engine SDK. Specifically:_

- The documentation meant for SDK devs, not client devs utilizing the SDK, has been moved from storybook to the readme. Some of this documentation (for example, how to publish), was in both places, so it was consolidated in the process.
- The alt text for images was referencing a client, so their name has been removed and accurate alt text used instead
- The arc logo, while part of the SDK, is not meant for client use. The documentation has been kept, but it's been moved to an internal folder where it will not be build as part of the ALC process.

## Jira Ticket
- [PEN-1237](https://arcpublishing.atlassian.net/browse/PEN-1237)

## Test Steps
- Run `npm run storybook`
- Confirm listed changes have been made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,36 +17,79 @@ In the future, this will be hosted. To build, see [documentation](https://storyb
...
 2. Branch off the `staging` branch
 
-`git checkout -b PEN-[jira ticket num]-[brief description of feature]`
+    - `git checkout -b PEN-[jira ticket num]-[brief description of feature]`
 
-3. Do the work (heh). Commit as you go, which will run the linter and tests.
+3. Do the work, commit as you go, which will run the linter and tests.
 
-4. Make pull request using GitHub against the `staging` branch. Get approval for your pr on your feature branch. 
+4. Make pull request using GitHub against the   staging` branch.
```
Comment (JackHowa) [inline README.md:32]:
```suggestion
4. Make pull request using GitHub against the `staging` branch. 
```

**Comment by RobinGiannattasio** [comment]
Good call - I just chose master because it seemed to line up much more accurately with what was on ALC. I'll go ahead and get this branch caught up with staging and then change where the PR is pointing to.

**Comment by JackHowa** [comment]
No worries ^

**Comment by JackHowa** [comment]
https://www.chromatic.com/snapshot?appId=5eed0506faad4f0022fedf95&id=5f2b085e41c26c00228f75e5 -> looks like the string changes in the storybook is asking to be reviewed https://github.com/WPMedia/engine-theme-sdk/pull/76/files#diff-82aeb7687c563c78fce0e6883993b4f8R155

**Comment by JackHowa** [comment]
good to go!

**Comment by RobinGiannattasio** [comment]
> https://www.chromatic.com/snapshot?appId=5eed0506faad4f0022fedf95&id=5f2b085e41c26c00228f75e5 -> looks like the string changes in the storybook is asking to be reviewed https://github.com/WPMedia/engine-theme-sdk/pull/76/files#diff-82aeb7687c563c78fce0e6883993b4f8R155

Gotcha - is that something I can approve? Since this is a desired change? I'm not sure what the next steps are here :D

**Comment by JackHowa** [comment]
oh it was just a small thing ... different strings used in the storybook for captions. if it's a deliberate or merge change, I don't think it's a massive deal. but if it had been a big change in storybook, you'd want to have a discussion about it

File: stories/Image.stories.mdx [hunk 2]
```diff
@@ -56,7 +56,7 @@ import { Image } from '@wpmedia/engine-theme-sdk';
             <p>Scroll down to see lazy loading example</p>
             <Image
                 url={'https://arc-anglerfish-arc2-prod-corecomponents.s3.amazonaws.com/public/37UMUNYNOVCEJDZW5SBKBXNMO4.jpg'}
-                alt={'This is a Free article for testing Mentor Medier’s paywall'}
+                alt={'Aerial view of bridge'}
```
Comment (JackHowa) [inline stories/Image.stories.mdx:59]: this was just the small alt tags that changed and caused storybook to think there had been a real code change

**Pull request merged**